pom.xml

```diff
@@ -132,9 +132,9 @@
                                 <dirset dir="${project.build.directory}/generated-sources/hawtjni/lib/META-INF/native/" id="platform">
                                     <include name="*" />
                                 </dirset>
-                                <copy file="${project.basedir}/../yara"
+                                <copy file="${env.YARA_HOME}/yara"
                                       tofile="${project.build.directory}/test-classes/META-INF/native/${toString:platform}/yara"/>
-                                <copy file="${project.basedir}/../yarac"
+                                <copy file="${env.YARA_HOME}/yarac"
                                       tofile="${project.build.directory}/test-classes/META-INF/native/${toString:platform}/yarac"/>
                             </tasks>
                         </configuration>
```

Comment:
Instead of using directly "env.YARA_HOME" let's use a property "${yara_home}" with the default value "${project.basedir}/.." if "env.YARA_HOME" is not set. This way both ways of compiling still hold.